### DISCUSSION TITLE
Castellation refinement Failed [1]

### DISCUSSION THREAD

**Discussion by SergioP1975** [nextfoam/baram#139] [Q&A]
I´m trying to mesh a complex part, but when it comes to the Castellation step, I cannot make it work. Have tried with and without refinement and the error message is the same.

Attached a link to the model and STL

https://drive.google.com/file/d/1PDNsoVMFP6dI_XDV8YxBslqOTal8DmNn/view?usp=sharing

In the console there is no much information, only 

C:\PROGRAMAS\Baram\solvers\openfoam\bin\snappyHexMesh aborted the job. abort code 1
---- error analysis -----

**Comment by kdg5424** [comment]
Hello @SergioP1975. This is Donggyu from NEXTFOAM.
The reason why the problem happens was because of the name of the geometry.
In console, there was a comment like this.
![image](https://github.com/user-attachments/assets/df30bec0-21b5-46c4-b21b-6cb15c94b3b3)
And I suspected the name of the geometry : It consists of only number.
So I changed the name of the geometry as "Pipe". And it works very well.
I wish you to try again with attach file. I change only name.

Best regards

Donggyu Kim
[pipe.zip](https://github.com/user-attachments/files/17551337/pipe.zip)

**Reply by SergioP1975** [reply]
Thanks a lot for your valuable help @kdg5424 !!!

Best regards
